dnf/conf/config.py

```diff
@@ -848,6 +849,15 @@ def _configure_from_options(self, opts):
                     msg = "Main config did not have a %s attr. before setopt"
                     logger.warning(msg, name)
 
+    def _adjust_conf_options(self):
+        """Adjust conf options interactions"""
+
+        skip_broken = self._get_option('skip_broken')
+        skip_broken_val = skip_broken._get()
```

Comment:
I am not sure about this, but isn't it same as `skip_broken`? I mean isn't this `skip_broken` equal to `skip_broken_val`?